client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-select-checkbox/index.tsx

```diff
@@ -8,29 +10,69 @@ interface Props {
 	item: SiteData;
 	siteError: boolean;
 	isLargeScreen?: boolean;
+	disabled?: boolean;
+	tooltip?: string;
 }
 
-export default function SiteSelectCheckbox( { item, siteError, isLargeScreen }: Props ) {
+export default function SiteSelectCheckbox( {
+	item,
+	siteError,
+	isLargeScreen,
+	disabled,
+	tooltip,
+}: Props ) {
+	const [ showTooltip, setShowTooltip ] = useState( false );
+	const checkboxRef = useRef< HTMLSpanElement | null >( null );
+
 	const recordEvent = useJetpackAgencyDashboardRecordTrackEvent(
 		[ item.site.value ],
 		isLargeScreen
 	);
 
-	function handleCheckboxClick() {
+	const handleCheckboxClick = () => {
 		item.onSelect?.();
 		recordEvent( item.isSelected ? 'site_unselected' : 'site_selected' );
-	}
+	};
+
+	const handleShowTooltip = () => {
+		setShowTooltip( true );
+	};
+
+	const handleHideTooltip = () => {
+		setShowTooltip( false );
+	};
 
 	return (
-		<span className="site-select-checkbox">
-			<FormInputCheckbox
-				className="disable-card-expand"
-				id={ `${ item.site.value.blog_id }` }
-				onClick={ handleCheckboxClick }
-				checked={ item.isSelected }
-				readOnly={ true }
-				disabled={ siteError }
-			/>
-		</span>
+		<>
```

Comment:
I don't think we need the empty tags here, or does the tooltip need to be outside the span(may be that's why you needed the margin-top here?

Reply:
Good catch. Forgot to clean this when I moved some of the components. 👍